src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3947,6 +3951,15 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         SearchMissingCommand.raise(*this, args);
     }
 
+    til::CoordType TermControl::_searchScrollOffset() const

```

Comment:
kinda seems weird that we have to recalculate this every time, even if the font size doesn't change. 

I know it doesn't currently, but I suppose it makes enough sense that the search box might get taller. But the DPI and font size don't change frequently. 

I feel like there's gotta be a way to just figure this out once per font size change (which I think also does happen on a dpi change) and just cache that value...

Reply:
updated to only be calculated in _coreFontSizeChanged.  Testing locally it got called every time I changed the font size or the DPI.  It looked like the scrolling also got updated afterwards by _coreOutputIdle